aozorabank/transfer.go

```diff
@@ -195,7 +197,7 @@ func (cli *Client) TransferRequest(
 		return nil, err
 	}
 	header := getTransferHeader(req.AccessToken)
-	header.Set(IdempotencyKeyHeaderKey, req.AccountID)
+	header.Set(IdempotencyKeyHeaderKey, uuid.NewString())
```

Comment:
IdempotencyKeyHeaderKey(冪等キー)はリクエストごとに発行する必要があるため修正

Reply:
すみません。
こちら、デグレっていた様ですが正しくは↓が正しそうです。

```suggestion
	header.Set(IdempotencyKeyHeaderKey, req.IdempotencyKey)
```

冪等キーは発行元のを利用して、アプリケーション側でも管理できた方が良いため。

Reply:
ありがとうございます。
修正しました。
[54b46bd](https://github.com/matopenKW/go-gmo/pull/2/commits/54b46bd61d0e4bf58f3460d667cc191f82e41469)